nuxt.config.ts

```diff
@@ -80,6 +80,8 @@ export default defineNuxtConfig({
           '\'self\'',
           'blob:',
         ],
+        // NOTE: Resolve Safari force HTTPS in development
+        'upgrade-insecure-requests': process.env.NODE_ENV === 'development' ? false : true,
```

Comment:
Who set `NODE_ENV`? Would `!== 'production'` be more robust?

Reply:
I think this should be always turned on for other cases
Ref: https://nuxt-security.vercel.app/advanced/faq#running-app-with-host-flag